remove LEAF_TYPEs for conifers

jgrenadier opened the pull request:
fixed in species.json

jgrenadier committed:
remove LEAF_TYPEs for conifers

fixed in species.json

TreeMama commented on android/local.properties:
@bestw07 should we be commiting android settings in the repo >?

bestw07 replied:
No, it doesn't need to be committed. It's its own environment setting for each developer. If you want, I can add it to the .gitignore file.

jgrenadier committed:
Update species.json

upgrade to most recent species.json and then made the changes necessary for fix the conifer tree leaf_type

bestw07 committed:
Delete local.properties

jgrenadier commented:
upgraded to species.json from develop and refixed the conifers.
Saw an exception after adding trees but I dont think this is related to this simple species.json change.
I'll add more details tomorrow.

bestw07 commented:
ok, sure, @jgrenadier

TreeMama reviewed (changes requested):
sorry - the file needs to be sync'd with develop again! almost there though!
